### ISSUE TITLE
release mode: build fails when a previous build succeeded

### ISSUE THREAD

**Issue by softlion** [xamarin/xamarin-android#1867]
### Steps to Reproduce

1. Create an android app, check "generate one package per selected ABI"
2. build it in release mode, deploy it on a real device (CTRL-F5)
3. change a comment in a .cs source file, and again build and deploy it on a real device (CTRL-F5)

### Expected Behavior

It build/deploys fine.

### Actual Behavior

It fails on zipalign.

### Version Information

VS windows 15.7.4

### Log File
```
3>C:\Dev\android-sdk\build-tools\27.0.3\zipalign.exe 4 "C:\Dev\repos\app\MyApp\MyApp.Droid\obj\Release\android\bin\com.myapp.myapp-arm64-v8a.apk" "bin\Release\\com.myapp.myapp-arm64-v8a-Signed.apk" 
3>Output file 'bin\Release\\com.myapp.myapp-arm64-v8a-Signed.apk' exists
3>"zipalign.exe" exited with code 1.
3>Done executing task "AndroidZipAlign" -- FAILED.
3>Done building target "_Sign" in project "MyApp.Droid.csproj" -- FAILED.
3>Done building project "MyApp.Droid.csproj" -- FAILED.
3>Build FAILED.
3>
3>Deploy failed on HUAWEI BLN-L21
```

> VS bug [#636634](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/636634)

**Comment by jonathanpeppers** [comment]
@softlion can you post a [diagnostic build log](https://docs.microsoft.com/en-us/xamarin/android/troubleshooting/troubleshooting#diagnostic-msbuild-output) for this project? The log you have doesn't actually contain the error message.

I will try to repro with a "hello world" project.

**Comment by jonathanpeppers** [comment]
Ok, I can repro it and post logs.

1. `File | New Project` I picked Android 8.1
1. Switched to `Release` config, enabled the `package per abi` option
1. Enabled all Android ABIs
1. Deploy with `msbuild MyApp.csproj /t:Install /p:Configuration=Release /bl:First.binlog`
1. Make a C# code change
1. Deploy with `msbuild MyApp.csproj /t:Install /p:Configuration=Release /bl:Second.binlog`

The error message is:
```
"App10.csproj" (Install target) (1) ->
(_Sign target) ->
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\Xamarin\Android\Xamarin.Android.Common.targets(2599,2): error MSB6006: "zipalign.exe" exited with code 1. [App10
.csproj]

    17 Warning(s)
    1 Error(s)
```

Attached binlogs: [logs.zip](https://github.com/xamarin/xamarin-android/files/2124153/logs.zip)

**Comment by dellis1972** [comment]
@jonathanpeppers this looks related to [1]


[1] https://github.com/xamarin/xamarin-android/commit/41bc8ffa0b8f21c4b41cc33fb629b433cfc5b3fd

**Comment by dellis1972** [comment]
Ah the .targets in the binlog do not have that fix. That will be why its not working. Looks like that fix did not make it into 15.7.4. It should be in 15.8.* though.

**Comment by dellis1972** [comment]
The work around for now is to set `<AndroidUseApkSigner>False</AndroidUseApkSigner>` in your csproj

**Comment by dellis1972** [comment]
Do this is a dup of #1519

**Comment by jonathanpeppers** [comment]
@softlion I'm going to close this for now, as I think this is fixed in 15.8.

Give the current preview a try and let us know!
